CQUI.modinfo

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="4.0">
+<Mod id="1d44b5e7-753e-405b-af24-5ee634ec8a01" version="1.0">
```

Comment:
what happens when you reset version from 4 to 1?
I am not familiar with Steam workshop mods at all, but my general **expectation** for a system that keeps downloadable content would be:
* a mod with a certain ID cannot be overwritten by people without permissions
* re-uploading a mod with the same ID and a previously uploaded version should not be possible (should be HTTP 400 - user error, if they use REST API)
* if someone other than the author wants to upload a mod with similar content, they will have to use a different ID.

if you were able to upload this mod to Steam with the previously used ID, and even reuse some version ("1.0") that most likely has been used before, it would suggest that none of those expectations are valid. being able to overwrite someone else's mod in Steam sounds like a huge security hole (at the very least) to me... weird.